terraso_backend/apps/shared_data/visualization_tileset_tasks.py

```diff
@@ -0,0 +1,161 @@
+import csv
+import json
+import threading
+
+import openpyxl
+import structlog
+from django.conf import settings
+
+from apps.core.gis.mapbox import create_tileset, remove_tileset
+from apps.shared_data.services import data_entry_upload_service
+
+from .models import VisualizationConfig
+
+logger = structlog.get_logger(__name__)
+
+
+class AsyncTaskHandler:
+    def start_task(self, method, args):
+        t = threading.Thread(target=method, args=[*args], daemon=True)
```

Comment:
question: Is there an advantage to setting `daemon=True` here? Looking at the Python [thread docs](https://docs.python.org/3/library/threading.html#thread-objects), it only says that it affects whether the Python program will exit or not.

This is mostly for my own knowledge. I was wondering if maybe you had to set the threads as daemon threads to make something work.

Reply:
@david-code I am not sure if this will have an impact on how our service runs since my expectation is that on a redeploy or a restart of a service render will not be blocked by the program begin able to finish or not. But, I think from the definition of a daemon thread it makes more sense for this case, I don't think we want to make the threads executing the creation of the tilesets a critical part of the process, we could change that in the future but I think for now it makes sense for them to be threads that will not block the program exit, let me know what do you think

Reply:
@david-code Hey David let me know what do you think about this